registry-library.adoc

```diff
@@ -0,0 +1,53 @@
+= Devfile registry library
+
+== Overview
+Devfile registry library is used for interacting with devfile registry, consumers can use devfile registry library to list stacks and/or samples of devfile registry, download the stack devfile and the whole stack from devfile registry.
+
+== What's included
```

Comment:
I don't think we necessarily need to include this section since this is more for developing the registry library (rather than consuming it). It's already in the readme too.

Reply:
You are right, I am gonna remove the `What's included` section.